assets/src/api/api.ts

```diff
@@ -33,16 +52,20 @@ export const RoomApiAxiosParamCreator = function (configuration?: Configuration)
         /**
          * Create a new peer in a room and get its token
          * @summary Join a room
-         * @param {string} roomName Room name
+         * @param {string} roomId Room ID
+         * @param {string} userId User ID
          * @param {any} [body] Room params
          * @param {*} [options] Override http request option.
          * @throws {RequiredError}
          */
-        videoroomWebRoomControllerShow: async (roomName: string, body?: any, options: RawAxiosRequestConfig = {}): Promise<RequestArgs> => {
-            // verify required parameter 'roomName' is not null or undefined
-            assertParamExists('videoroomWebRoomControllerShow', 'roomName', roomName)
-            const localVarPath = `/api/room/{room_name}`
-                .replace(`{${"room_name"}}`, encodeURIComponent(String(roomName)));
+        apiRoomsRoomIdUsersUserIdGet: async (roomId: string, userId: string, body?: any, options: RawAxiosRequestConfig = {}): Promise<RequestArgs> => {
+            // verify required parameter 'roomId' is not null or undefined
+            assertParamExists('apiRoomsRoomIdUsersUserIdGet', 'roomId', roomId)
+            // verify required parameter 'userId' is not null or undefined
+            assertParamExists('apiRoomsRoomIdUsersUserIdGet', 'userId', userId)
+            const localVarPath = `/api/rooms/{roomId}/users/{userId}`
+                .replace(`{${"roomId"}}`, encodeURIComponent(String(roomId)))
+                .replace(`{${"userId"}}`, encodeURIComponent(String(userId)));
```

Comment:
Looks like this is outdated, room manager uses `roomName` (here and in other places in this file)